std/concurrency.d

```diff
@@ -2473,6 +2474,23 @@ private
 
 version( unittest )
 {
+    void changeScheduler(Scheduler s)
+    {
+        import core.memory : GC;
+        import core.thread : Thread;
+
+        bool dosleep;
+        while (Thread.getAll().length != 1)
+        {
+            if (dosleep)
+                Thread.sleep(dur!("msecs")( 10 ));
+            else
+                dosleep = true;
+            GC.collect;
```

Comment:
A comment would be nice to explain why you are doing a collection as it's not immediately obvious.